rust/src/transformations/quantile_score_candidates/make_quantile_score_candidates.tex

```diff
@@ -0,0 +1,318 @@
+    &\le \alpha_{den} \cdot \max\limits_{i} ||\#(X < C_i) - \alpha \cdot (|X| - \#(X = C_i))|
+        \\&\hspace{1cm}|\#(X' < C_i) - \alpha \cdot (|X| - \#(X' = C_i))|| \\
+\intertext{Consider each of the three cases of adding or removing an element in X.}
+\intertext{Case 1. Assume $X'$ is equal to $X$, but with some $X_j < C_i$ added or removed.}
+    &= \alpha_{den} \cdot \max\limits_{i} ||(1 - \alpha) \cdot \#(X < C_i) - \alpha \cdot \#(X > C_i)| 
+        \\&\hspace{1cm} - |(1 - \alpha) \cdot (\#(X < C_i) \pm 1) - \alpha \cdot \#(X > C_i)|| \\
+    &\leq \alpha_{den} \cdot  \max\limits_{i} ||(1 - \alpha) \cdot \#(X < C_i) - \alpha \cdot \#(X > C_i)|
+        \\&\hspace{1cm} - (|(1 - \alpha) \cdot \#(X < C_i) - \alpha \cdot \#(X > C_i)| + |\pm(1 - \alpha)|)| &&\text{by triangle inequality} \\
+    &= \alpha_{den} \cdot \max\limits_{i} |1 - \alpha| &&\text{scores cancel}\\
+    &= \alpha_{den} - \alpha_{num} &&\text{since } \alpha \leq 1
+\intertext{Case 2. Assume $X'$ is equal to $X$, but with some $X_j > C_i$ added or removed.}
+    &= \alpha_{den} \cdot \max\limits_{i} ||(1 - \alpha) \cdot \#(X < C_i) - \alpha \cdot \#(X > C_i)| 
+        \\&\hspace{1cm} - |(1 - \alpha) \cdot \#(X < C_i) - \alpha \cdot (\#(X > C_i) \pm 1)|| \\
+    &\leq \alpha_{den} \cdot \max\limits_{i} ||(1 - \alpha) \cdot \#(X < C_i) - \alpha \cdot \#(X > C_i)| 
+        \\&\hspace{1cm} - (|(1 - \alpha) \cdot \#(X < C_i) - \alpha \cdot \#(X > C_i)| + |\pm\alpha|)| &&\text{by triangle inequality} \\
+    &= \alpha_{den} \cdot \max\limits_{i} |\alpha| &&\text{scores cancel}\\
+    &= \alpha_{num} &&\text{since } \alpha \geq 0
+\intertext{Case 3. Assume $X'$ is equal to $X$, but with some $X_j = C_i$ added or removed.}
+    &= \alpha_{den} \cdot \max\limits_{i} ||(1 - \alpha) \cdot \#(X < C_i) - \alpha \cdot \#(X > C_i)| 
+        \\&\hspace{1cm} - |(1 - \alpha) \cdot \#(X < C_i) - \alpha \cdot \#(X > C_i)|| \\
+    &= 0 &&\text{no change in score}
+\intertext{Take the union bound over all cases.}
+    &\le \max(\alpha_{num}, \alpha_{den} - \alpha_{num})
+\end{align*}
+
+\end{proof}
+
+Take any two elements $X, X'$ in the \texttt{input\_domain} and any pair $(\din, \dout)$, 
+where \din\ has the associated type for \texttt{input\_metric} and \dout\ has the associated type for \texttt{output\_metric}.
+Assume $X, X'$ are \din-close under \texttt{input\_metric} and that $\texttt{stability\_map}(\din) \leq \dout$. 
+
+\begin{align*}
+    \dout &= \max\limits_{X \sim X'} d_{IDD}(s, s') &&\text{where } s = \function(X)\\
+    &= \max\limits_{X \sim X'} \max\limits_{ij} |(s_i - s'_i) - (s_j - s'_j)| &&\text{by definition of }\rustdoc{metrics/struct}{LInfDiffDistance} \\
+    &\leq 2 \max\limits_{X \sim X'} \max\limits_{i} |s_i - s'_i| &&\text{$s$ is not monotonic; take looser bound}\\
+    &\leq 2 \sum_j^{\din} \max\limits_{Z_{j} \sim Z_{j+1}} \max\limits_{i} |s_{i,j} - s_{i,j+1}| &&\text{by path property where } d_{Sym}(Z_i, Z_{i+1}) = 1, X = Z_0 \text{ and } Z_\din = X' \\
```

Comment:
it is a detail but maybe both in same order 
X = Z_0 \text{ and }  X'= Z_\din
or
Z_0 = X \text{ and } Z_\din = X'